scripts/retry

```diff
@@ -1,14 +1,2 @@
-#!/bin/bash
-RETRIES=${RETRIES:-20}
-PAUSE=${PAUSE:-10}
-
-for n in $(seq ${RETRIES}); do
-  echo "[${n}/${RETRIES}] ${*}"
-  if eval "${*}"; then
-    echo "[SUCC] ${n}/${RETRIES}"
-    exit 0
-  fi
-  sleep ${PAUSE}
-  echo "[FAIL] ${n}/${RETRIES}"
-done
-exit 1
+#!/bin/sh
+exec "$@"
```

Comment:
Why is the retry logic being removed here?

Reply:
Can restore this for final PR. While it's draft/in progress I prefer it to fail without long retry.

Reply:
OK, makes sense.